src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -148,20 +208,32 @@ export default App.extend({
       this.setState({ filters: _.clone(attributes) });
     });
   },
-  beforeStart() {
-    const filter = this.getState().getEntityFilter();
-    return Radio.request('entities', `fetch:${ this.getState().getType() }:collection`, { filter });
-  },
-  onStart(options, collection) {
-    this.collection = collection;
+  showMultiEditButtonView() {
+    this.getChildApp('filters').stop();
```

Comment:
I think this stop could move to the function that switches between these two things.